From 42d943f40a67fdbefe0bc9d445145969d6e5c8a0 Mon Sep 17 00:00:00 2001
From: David Bauer <mail@david-bauer.net>
Date: Thu, 18 Mar 2021 00:56:02 +0100
Subject: [PATCH] mediatek: fix broken UniFi 6 LR image

Ubiquiti's own bootloader expects the configuration mode to be present
with a "@" instead of a "-" for the sperator character. Otherwise
booting of the image fails.

Signed-off-by: David Bauer <mail@david-bauer.net>
---
 target/linux/mediatek/image/mt7622.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/mediatek/image/mt7622.mk b/target/linux/mediatek/image/mt7622.mk
index fd043ed85f..88d71600b2 100644
--- a/target/linux/mediatek/image/mt7622.mk
+++ b/target/linux/mediatek/image/mt7622.mk
@@ -62,6 +62,7 @@ TARGET_DEVICES += mediatek_mt7622-ubi
 define Device/ubnt_unifi-6-lr
   DEVICE_VENDOR := Ubiquiti
   DEVICE_MODEL := UniFi 6 LR
+  DEVICE_DTS_CONFIG := config@1
   DEVICE_DTS := mt7622-ubnt-unifi-6-lr
   DEVICE_DTS_DIR := $(DTS_DIR)/mediatek
   DEVICE_PACKAGES := kmod-mt7915e
-- 
2.27.0

